pkg/defaults/policies/files/cvss_6_privileged.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "93f4b2dd-ef5a-419e-8371-38aed480fb36",
   "name": "Fixable CVSS >= 6 and Privileged",
```

Comment:
Should we align the names as well? Compare `Fixable CVSS >= 6 and Privileged` to `Privileged Containers with Important and Critical Fixable CVEs`. Perhaps `Privileged Containers with CVSS >= 6 and Fixable CVEs` would make it more obvious that these are alternative choices?

Reply:
Yeah, looks like policy migrator looks at IDs so we can update the name as any other field.